src/components/Footer.astro

```diff
@@ -15,9 +15,9 @@
         <div class="my-4 lg:my-16 text-center lg:text-left">
             <h2 class="font-display uppercase mb-2">Contact</h2>
             <p>Hours: 2AM - 4PM EST</p>
-            <p>Email: hello@indevitus.com</p>
+            <p>hello@indevitus.com</p>
             <p class="mt-2">
-                <a class="inline-block" href="https://www.linkedin.com/company/indevitus/" title="LinkedIn">
+                <a class="inline-block hover:text-gray-light transition ease-in-out duration-300" target="_blank" href="https://www.linkedin.com/company/indevitus/" title="LinkedIn">
```

Comment:
Was this not done already in the other pull request?

Reply:
Hey @mpandzo we have the same changes in the tailwind config, but different element have this hover transition. 
Once we merge develop into this branch we will only have this hover transition on LinkedIn icon.